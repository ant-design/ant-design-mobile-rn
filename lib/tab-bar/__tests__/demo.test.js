import rnDemoTest from '../../../tests/shared/demoTest';
import SafeAreaView from '../__mocks__/react-native-safe-area-view';

jest.mock('react-native-safe-area-view', () => SafeAreaView);

rnDemoTest('tab-bar');
