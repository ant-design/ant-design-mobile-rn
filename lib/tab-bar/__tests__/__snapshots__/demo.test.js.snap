// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ./components/tab-bar/demo/basic.tsx correctly 1`] = `
<View
  forceInset={
    Object {
      "bottom": "always",
      "top": "never",
    }
  }
  style={
    Array [
      Object {
        "flex": 1,
      },
      Object {
        "backgroundColor": "#f5f5f5",
      },
    ]
  }
>
  <View
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <View
      style={
        Object {
          "flex": 1,
        }
      }
    >
      <View
        style={
          Array [
            Object {
              "backgroundColor": "white",
              "bottom": 0,
              "height": 0,
              "left": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
            },
            Object {
              "height": undefined,
            },
          ]
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "white",
              "flex": 1,
            }
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "#efeff4",
                "flexDirection": "row",
                "height": 44,
                "paddingLeft": 8,
                "paddingRight": 8,
              }
            }
          >
            <View
              style={
                Object {
                  "flex": 1,
                  "flexDirection": "row",
                }
              }
            >
              <TextInput
                allowFontScaling={true}
                clearButtonMode="always"
                editable={true}
                onBlur={[Function]}
                onChangeText={[Function]}
                onClear={[Function]}
                onFocus={[Function]}
                onSubmit={[Function]}
                onSubmitEditing={[Function]}
                placeholder="Search"
                style={
                  Array [
                    Object {
                      "backgroundColor": "#fff",
                      "borderColor": "#dddddd",
                      "borderRadius": 5,
                      "borderWidth": 0.5,
                      "color": "#000000",
                      "flex": 1,
                      "fontSize": 14,
                      "height": 28,
                      "paddingBottom": 0,
                      "paddingLeft": 35,
                      "paddingRight": 35,
                      "paddingTop": 0,
                    },
                    undefined,
                  ]
                }
                underlineColorAndroid="transparent"
                value=""
              />
            </View>
            <Text
              style={
                Array [
                  Object {
                    "color": "#cccccc",
                    "fontFamily": "antoutline",
                    "fontSize": 22,
                    "fontStyle": "normal",
                    "fontWeight": "normal",
                  },
                  Object {
                    "color": "#cccccc",
                    "fontSize": 15,
                    "left": 16,
                    "position": "absolute",
                    "top": 14.5,
                  },
                ]
              }
            >
              
            </Text>
            <View
              style={
                Object {
                  "alignItems": "center",
                  "height": 28,
                  "justifyContent": "center",
                }
              }
            >
              <Text
                onPress={[Function]}
                style={
                  Object {
                    "color": "#108ee9",
                    "fontSize": 16,
                    "paddingLeft": 15,
                  }
                }
              >
                取消
              </Text>
            </View>
          </View>
          <Text
            style={
              Object {
                "margin": 50,
              }
            }
          >
            Koubei Tab
          </Text>
        </View>
      </View>
    </View>
    <View
      style={
        Array [
          Object {
            "backgroundColor": "#f5f5f5",
          },
          Object {
            "borderColor": "#dddddd",
            "borderStyle": "solid",
            "borderTopWidth": 1,
            "flexDirection": "row",
            "height": 50,
          },
        ]
      }
    >
      <View
        accessible={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "alignItems": "center",
              "flex": 1,
              "justifyContent": "center",
            },
            null,
          ]
        }
      >
        <View>
          <Text
            style={
              Array [
                Object {
                  "color": "#949494",
                  "fontFamily": "antoutline",
                  "fontSize": 22,
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
                undefined,
              ]
            }
          >
            
          </Text>
        </View>
        <Text
          style={
            Array [
              Object {
                "fontSize": 10,
                "marginTop": 2,
              },
              Object {
                "color": "#949494",
              },
            ]
          }
        >
          Life
        </Text>
      </View>
      <View
        accessible={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "alignItems": "center",
              "flex": 1,
              "justifyContent": "center",
            },
            Object {},
          ]
        }
      >
        <View>
          <Text
            style={
              Array [
                Object {
                  "color": "#33A3F4",
                  "fontFamily": "antoutline",
                  "fontSize": 22,
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
                undefined,
              ]
            }
          >
            
          </Text>
          <View
            style={
              Array [
                Object {
                  "backgroundColor": "#ff5b05",
                  "borderRadius": 10,
                  "height": 20,
                  "left": 20,
                  "minWidth": 20,
                  "paddingHorizontal": 5,
                  "position": "absolute",
                  "top": 0,
                },
              ]
            }
          >
            <Text
              style={
                Array [
                  Object {
                    "color": "white",
                    "textAlign": "center",
                  },
                ]
              }
            >
              2
            </Text>
          </View>
        </View>
        <Text
          style={
            Array [
              Object {
                "fontSize": 10,
                "marginTop": 2,
              },
              Object {
                "color": "#33A3F4",
              },
            ]
          }
        >
          Koubei
        </Text>
      </View>
      <View
        accessible={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "alignItems": "center",
              "flex": 1,
              "justifyContent": "center",
            },
            null,
          ]
        }
      >
        <View>
          <Text
            style={
              Array [
                Object {
                  "color": "#949494",
                  "fontFamily": "antoutline",
                  "fontSize": 22,
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
                undefined,
              ]
            }
          >
            
          </Text>
        </View>
        <Text
          style={
            Array [
              Object {
                "fontSize": 10,
                "marginTop": 2,
              },
              Object {
                "color": "#949494",
              },
            ]
          }
        >
          Friend
        </Text>
      </View>
      <View
        accessible={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "alignItems": "center",
              "flex": 1,
              "justifyContent": "center",
            },
            null,
          ]
        }
      >
        <View>
          <Text
            style={
              Array [
                Object {
                  "color": "#949494",
                  "fontFamily": "antoutline",
                  "fontSize": 22,
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
                undefined,
              ]
            }
          >
            
          </Text>
        </View>
        <Text
          style={
            Array [
              Object {
                "fontSize": 10,
                "marginTop": 2,
              },
              Object {
                "color": "#949494",
              },
            ]
          }
        >
          My
        </Text>
      </View>
    </View>
  </View>
</View>
`;
